agent/07_agent_add_node.sh

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env bash
+set -euxo pipefail
+
+SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )/.." && pwd )"
+
+source $SCRIPTDIR/common.sh
+source $SCRIPTDIR/network.sh
+source $SCRIPTDIR/ocp_install_env.sh
+source $SCRIPTDIR/utils.sh
+source $SCRIPTDIR/validation.sh
+source $SCRIPTDIR/agent/common.sh
+
+early_deploy_validation
+
+function approve_csrs() {
+  # approve CSRs for up to 30 mins
+  timeout=$((30*60))
+  elapsed=0
+  while (( elapsed < timeout )); do
+    pending_csrs=$(oc get csr | grep Pending)
+    if [[ ${pending_csrs} != "" ]]; then
+      echo "Approving CSRs: $pending_csrs"
+      echo $pending_csrs | cut -d ' ' -f 1 | xargs oc adm certificate approve
+    fi
+    elapsed=$((elapsed + 10))
+    sleep 10
+  done
+}
+
+if [ -f node-joiner.sh ]; then
+  rm -f node-joiner.sh
+fi
+if [ -f node.x86_64.iso ]; then
+  rm -f node.x86_64.iso
+fi
+wget https://raw.githubusercontent.com/openshift/installer/master/docs/user/agent/add-node/node-joiner.sh
```

Comment:
Could you please download the scripts (and generate the ISO) within the `${SCRIPTDIR}/${OCP_DIR}/add-node/` folder? Otherwise they will appear as git changes

Reply:
Sure.